lib/types/types.ex

```diff
@@ -46,6 +46,4 @@ defmodule Types do
   @type blob_index :: uint64
   # Max size: BYTES_PER_FIELD_ELEMENT * FIELD_ELEMENTS_PER_BLOB
   @type blob :: binary
-  @type kzg_commitment :: bytes48
-  @type kzg_proof :: bytes48
```

Comment:
The dialyzer errors are due to these two guys dissapearing. Let's keep them, but as a reference to `Kzg` types (i.e. `@type kzg_proof :: Kzg.proof()`). Same for Bls.